src/views/Root/Root.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import './index.css';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import AppContext from '../../context';
+import Modal from '../../components/Modal/Modal';
+import Header from '../../components/Header/Header';
+import GroceryListView from '../GroceryListView/GroceryListView';
+import RecipesListView from '../RecipesListView/RecipesListView';
+import StatisticsView from '../StatisticsView/StatisticsView';
+import Form from '../../components/Form/Form';
+import FormTypes from '../../components/Form/FormTypes';
+import { GroceryService } from './GroceryService';
+
+class Root extends Component {
+	state = {
+		isModalOpen: false,
+		formType: FormTypes.addItem,
+		grocery: [],
+		searchQuery: '',
+		groceryCompleted: [],
+		itemUnderEdition: '',
+		isNameSearch: true,
+		showCompleted: true
+	};
+
+	groceryService = new GroceryService();
+
+	componentWillMount = () => {
```

Comment:
Be consistent in the way you declare methods. The form `componentWillMount()` is preferred over `componentWillMount = () => `